src/main/java/com/microsoftopentechnologies/windowsazurestorage/WAStoragePublisher.java

```diff
@@ -358,6 +341,54 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath ws, @Nonnull Launc
 		}
 	}
 
+	private void validateData(Run<?, ?> run,
+			TaskListener listener, StorageAccountInfo storageAccount,
+			String expContainerName) throws IOException, InterruptedException {
```

Comment:
@arroyc This diff is outdated. Validation of the container name is added in a later commit.